pyfolio/tears.py

```diff
@@ -950,8 +950,8 @@ def create_interesting_times_tear_sheet(
     if benchmark_rets is not None:
         returns = utils.clip_returns_to_benchmark(returns, benchmark_rets)
 
-    bmark_interesting = timeseries.extract_interesting_date_ranges(
-        benchmark_rets)
+        bmark_interesting = timeseries.extract_interesting_date_ranges(
+            benchmark_rets)
```

Comment:
Right! I think these two lines blame to me: it's probably something I overlooked in #536.

Reply:
Got fixed. Thanks